app/pods/search-schema/create/controller.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+    schemaName: '',
+    schemaContent: ''
+});
```

Comment:
This holds the temporary values of the new schema until they are ready to be sent to the server.